specification/network/resource-manager/Microsoft.Network/stable/2022-01-01/NetworkWatcherDiagnosticAPI

```diff
@@ -0,0 +1,405 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "NetworkWatcherDiagnosticApi",
+		"description": "The Microsoft Azure Network Watcher Diagnostic API provides a RESTful set of web services that interact with Microsoft Azure Network watcher service to check connectivity in your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Network Watcher service.",
+		"version": "2021-02-01"
+	},
+	"host": "",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security": [
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow.",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/api.diagnosticstest.msftcloudes.com/api/diagnosticdata/": {
+			"post": {
+				"tags": [
```

Comment:
are these tenant level apis ? Also these are in the wrong namespace .. The folder is the Microsoft.Network namespace. "api.diagnosticstest.msftcloudes.com" is not a valid namespace